web/src/components/filters/compare-filter.tsx

```diff
@@ -0,0 +1,66 @@
+import { Dropdown, DropdownItem, DropdownToggle, DropdownToggleAction } from '@patternfly/react-core';
+import * as React from 'react';
+import { useTranslation } from 'react-i18next';
+
+export enum FilterCompare {
+  EQUAL = 1,
+  NOT_EQUAL
+}
+
+export interface CompareFilterProps {
+  state: FilterCompare;
```

Comment:
nit: the word "state" is a bit confusing given what it means in React ... what about "value" ? or "operator" (this is a comparison operator, after all) ?

Reply:
done